src/stories/Editor.stories.tsx

```diff
@@ -11,53 +13,130 @@ const content = `
   <strong><span style="font-size: 14pt;"><span style="color: #7e8c8d; font-weight: 600;">No matter what you're building, TinyMCE has got you covered.</span></span></strong>
 </p>`;
 
+let lastChannel = '5';
+const getConf = (stringConf) => {
+  let conf  = {};
+  console.log('parsing: ', stringConf);
+  try {
+    conf = JSON.parse(stringConf);
+  } catch (err) {
+    console.error('failed to parse configuration: ', err);
+  }
+  return conf;
+}
+
+
 export default {
   title: 'Editor',
   component: Editor,
+  argTypes: {
+    channel: {
+      defaultValue: '5',
+      options: ['5', '5-dev', '6-dev', '6-testing', '6-stable'],
+      control: { type: 'select'}
+    },
+    conf: {
+      defaultValue: '{"height": 300}',
+      control: { type: 'text' }
+    }
+  },
   parameters: {
     controls: {
       hideNoControlsWarning: true
     }
   }
 };
 
-export const Iframe: Story = () => ({
+export const Iframe: Story = (args) => ({
   components: {Editor},
   setup() {
+    onBeforeMount(() => {
+      if (args.channel !== lastChannel) {
+        delete (window as any).tinymce;
```

Comment:
You'll likely also want to delete `window.tinyMCE` if you want to clean up all references.

Reply:
Deleting this as well